### PR TITLE
Support for standalone RevoScale package scenario - Linux

### PR DESCRIPTION
**Pull request by arunguru-msft**
Adding support for standalone packages

### CODE REVIEW

**Comment by jborsecnik** [comment]
@WilliamDAssafMSFT, 

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/linux/sql-server-linux-setup-machine-learning-sql-2022.md [hunk 1]
```diff
@@ -91,15 +93,10 @@ The following commands register the repository providing the R language platform
     install.packages("jsonlite", lib="/usr/lib/R/library")
     ```
 
-1. Download and install `CompatibilityAPI` for Linux.
+1. Download and install `CompatibilityAPI` amd `RevoScaleR` for Linux.
```
Comment (WilliamDAssafMSFT) [inline docs/linux/sql-server-linux-setup-machine-learning-sql-2022.md:96]:
```suggestion
1. Download and install `CompatibilityAPI` and `RevoScaleR` for Linux.
```

File: docs/linux/sql-server-linux-setup-machine-learning-sql-2022.md [hunk 2]
```diff
@@ -109,6 +106,7 @@ The following commands register the repository providing the R language platform
     library("RevoScaleR")
     ```
 
+### Configure R runtime with SQL Server
```
Comment (WilliamDAssafMSFT) [inline docs/linux/sql-server-linux-setup-machine-learning-sql-2022.md:109]:
```suggestion
### Configure R runtime with SQL Server

```

File: docs/linux/sql-server-linux-setup-machine-learning-sql-2022.md [hunk 3]
```diff
@@ -138,26 +136,8 @@ The following commands register the repository providing the R language platform
...
-    ```
-
-1. Review and accept the End User License Agreement (EULA) for SQL Server ML Services.
-
-    ```bash
-    sudo /opt/mssql/bin/mssql-conf set EULA accepteulaml Y
-    ```
-
+## Setup Python support
+### Install Python runtime
```
Comment (WilliamDAssafMSFT) [inline docs/linux/sql-server-linux-setup-machine-learning-sql-2022.md:140]:
```suggestion
## Setup Python support

### Install Python runtime

```

File: docs/linux/sql-server-linux-setup-machine-learning-sql-2022.md [hunk 4]
```diff
@@ -171,6 +151,7 @@ The following commands register the repository providing the R language platform
     ```python
     import revoscalepy
     ```
+### Configure Python runtime with SQL Server
```
Comment (WilliamDAssafMSFT) [inline docs/linux/sql-server-linux-setup-machine-learning-sql-2022.md:154]:
```suggestion

### Configure Python runtime with SQL Server
```

File: docs/linux/sql-server-linux-setup-machine-learning-sql-2022.md [hunk 5]
```diff
@@ -243,6 +224,9 @@ Follow the [Offline installation](sql-server-linux-setup.md#offline) instruction
 > [!Tip]
 > Several of the package management tools provide commands that can help you determine package dependencies. For yum, use `sudo yum deplist [package]`. For Ubuntu, use `sudo apt-get install --reinstall --download-only [package name]` followed by `dpkg -I [package name].deb`.
 
+## Standalone RevoScale packages for Python and R runtime
```
Comment (WilliamDAssafMSFT) [inline docs/linux/sql-server-linux-setup-machine-learning-sql-2022.md:227]:
```suggestion
## Standalone RevoScale packages for Python and R runtime

```

**Review by WilliamDAssafMSFT — CHANGES_REQUESTED** [review]
minor edits

**Review by WilliamDAssafMSFT — APPROVED** [review]

**Comment by WilliamDAssafMSFT** [comment]
#sign-off

**Pull request merged**